configuration/override_dir_structure.rst

```diff
@@ -61,20 +63,28 @@ In this code, ``$this->environment`` is the current environment (i.e. ``dev``).
 In this case you have changed the location of the cache directory to
 ``var/{environment}/cache/``.
 
+The second option is to use a predefined environment variable
+named ``APP_CACHE_DIR``. Its value must be the full path of the cache folder.
+
 .. caution::
 
     You should keep the cache directory different for each environment,
     otherwise some unexpected behavior may happen. Each environment generates
     its own cached configuration files, and so each needs its own directory to
     store those cache files.
 
+    When using ``APP_CACHE_DIR`` variable be sure you setup the
```

Comment:
Now that #37232 is merged, I guess we can remove this paragraph.

Reply:
yes depending on the decision I was ready to update ;) done!